unikraft/app/application.go

```diff
@@ -915,11 +916,16 @@ func (app application) WithTarget(targ target.Target) (Application, error) {
 // MarshalYAML makes application implement yaml.Marshaller
 func (app application) MarshalYAML() (interface{}, error) {
 	ret := map[string]interface{}{
-		"spec":     schema.SchemaVersionLatest,
 		"name":     app.name,
 		"unikraft": app.unikraft,
 	}
 
+	if app.kraftfile.config["specification"] != nil {
+		ret["specification"] = schema.SchemaVersionLatest
+	} else {
+		ret["spec"] = schema.SchemaVersionLatest
```

Comment:
```suggestion
		ret["specification"] = schema.SchemaVersionLatest
```

Reply:
Updated conditional block to the following to make it more consistent:
```
if app.kraftfile.config["specification"] != nil {
    ret["specification"] = app.kraftfile.config["specification"]
} else if app.kraftfile.config["spec"] != nil {
    ret["spec"] = app.kraftfile.config["spec"]
} else {
    ret["specification"] = schema.SchemaVersionLatest
}
```